Source/CarthageKit/Project.swift

```diff
@@ -609,6 +605,54 @@ private func filesInDirectory(directoryURL: NSURL, typeIdentifier: String) -> Si
 		}
 }
 
+/// Sends the URL for the Info.plist of the specified Framework.
+private func infoPlistForFramework(frameworkURL: NSURL) -> SignalProducer<NSURL, CarthageError> {
+	return filesInDirectory(frameworkURL, "com.apple.property-list")
+		|> filter { plist in plist.lastPathComponent == "Info.plist" }
```

Comment:
@neilpa The ReactiveCocoa iOS framework has `Info.plist` at the root, while the Mac framework has it within the `Resources` symlink. Should I check those two paths explicitly rather than a recursive search? As to your other comment, if you inspect the implementation of `filesInDirectory`, it skips nested frameworks.